spec/project_config_spec.rb

```diff
@@ -673,6 +714,11 @@
       it 'should return nil when provided variation key is invalid' do
         expect(config.get_variation_from_id('test_experiment', 'invalid_variation')).to eq(nil)
       end
+
+      it 'should return featureEnabled false when provided featureEnabled is null' do
```

Comment:
Here message should be 'should return variation having featureEnabled false when not provided' as you compare two variations not their properties.

Reply:
Done